bt_image_resizer.drush.inc

```diff
@@ -136,16 +137,20 @@ function bt_image_resizer_process_batch($mime, $width, $height, $limit, $directo
 
   $image = image_load($file->uri);
 
-  // Scale the image to width/height as requested.
-  if(isset($resize_type['width'])) {
-    image_scale($image, $resize_type['width']);
-  }
-  else {
-    image_scale($image, NULL, $resize_type['height']);
-  }
+  if(is_object($image)) {
```

Comment:
@ChrisSwainBTM  Do you know the condition in which this value can be a boolean? I'm curious to know if we are potentially masking another issue by performing this check.